lib/eol/api/pages/v1_0.rb

```diff
@@ -197,6 +197,10 @@ def self.get_data_objects(taxon_concept, options={})
             solr_search_params[:vetted_types] = ['trusted']
           elsif options[:vetted] == 2  # 2 = everything except untrusted
             solr_search_params[:vetted_types] = ['trusted', 'unreviewed']
+          elsif options[:vetted] == 3  # 3 = unreviewed
+            solr_search_params[:vetted_types] = ['unreviewed'] 
+          elsif options[:vetted] == 4  # 4 = untrusted
+            solr_search_params[:vetted_types] = ['untrusted'] 
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.